### PR TITLE
make is easy to configure this via inventory

### PR DESCRIPTION
**Pull request by Natkeeran**
# What does this Pull Request do?
Make the urls as variable so they can be configured more easily in the inventory.

# How should this be tested?
* Replace the role with the PR
* Vagrant up
* Verify the things get index in Fedora

# Interested parties
@Islandora-Devops/committers

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -33,13 +39,13 @@ alpaca_settings:
     settings:
       input.stream: activemq:topic:fedora
       triplestore.reindex.stream: activemq:queue:triplestore.reindex
-      triplestore.baseUrl: http://localhost:8080/bigdata/namespace/islandora/sparql
+      triplestore.baseUrl: "{{ alpaca_triplestore_base_url }}/namespace/islandora/sparql"
```
Comment (whikloj) [inline defaults/main.yml]: If we are going part way, why not remove the assumption that all triplestores will use the "islandora" namespace.

File: defaults/main.yml [hunk 2]
```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
 alpaca_karaf_user: karaf
 alpaca_local_mvn_path: /opt/maven/repo
 
+alpaca_triplestore_base_url: http://localhost:8080/bigdata
+alpaca_milliner_base_url: http://localhost:8000/milliner/
+alpaca_gemini_base_url: http://localhost:8000/gemini/
+alpaca_houdini_base_url: http://localhost:8000/houdini/convert
+alpaca_homarus_base_url: http://localhost:8000/homarus/convert
```
Comment (whikloj) [inline defaults/main.yml]: Feels like the name `..._base_url` might be incorrect for these last two.

File: defaults/main.yml [hunk 3]
```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
 alpaca_karaf_user: karaf
 alpaca_local_mvn_path: /opt/maven/repo
 
+alpaca_triplestore_base_url: http://localhost:8080/bigdata
```
Comment (whikloj) [inline defaults/main.yml:29]: Probably should be consistent on whether to end with a slash or not. Unless we can adjust to account for either

File: defaults/main.yml (hunk 2 above)
Comment (Natkeeran) [reply defaults/main.yml]: @whikloj endpoint?

**Comment by Natkeeran** [comment]
@whikloj 

Addressed your feedback. 

Triplestore url is used here as well: https://github.com/Islandora-Devops/ansible-role-blazegraph.  Will update that role as well.

**Comment by dannylamb** [comment]
`vagrant up`

**Review by dannylamb — APPROVED** [review]
Tested along with the other PRs, and this works as intended.  e.g. nothing broke but now we can use all these variables.

**Pull request merged**